crates/next_version/src/version_increment.rs

```diff
@@ -136,3 +158,80 @@ impl VersionIncrement {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    // Returns true when a commit's custom type matches the provided regex
+    #[test]
+    fn returns_true_for_matching_custom_type() {
+        use conventional_commit_parser::commit::{CommitType, ConventionalCommit};
+        use regex::Regex;
+
+        let regex = Regex::new(r"custom").unwrap();
+        let commits = vec![ConventionalCommit {
+            commit_type: CommitType::Custom("custom".to_string()),
+            is_breaking_change: false,
+            summary: "A custom commit".to_string(),
+            body: None,
+            scope: None,
+            footers: vec![],
+        }];
+
+        let result = is_there_a_custom_match(Some(&regex), &commits);
+        assert!(result);
```

Comment:
There is no need for an assignment here, this can be handled in a single line.

Reply:
done 32a74cd